gravity-forms/gw-conditional-logic-operator-does-not-contain.php

```diff
@@ -45,6 +45,11 @@ public function output_admin_inline_script() {
 				operators.does_not_contain = 'does not contain';
 				return operators;
 			} );
+
+			// Override the default GF function to add our custom operator.
+			function ruleNeedsTextValue( rule ) {
+				return ['does_not_contain','contains', 'starts_with', 'ends_with', '<', '>' ].indexOf ( rule.operator ) !== -1;
+			}
```

Comment:
```suggestion
			let origRuleNeedsTextValue = window.ruleNeedsTextValue;
			// Override the default GF function to add our custom operator.
			function ruleNeedsTextValue( rule ) {
			    let needsTextValue = origRuleNeedsTextValue( rule );
			    return needsTextValue || rule.operator.indexOf( 'does_not_contain' ) !== -1;
			}
```

Just to be good neighbors, I'd use this approach so that other people could override this function themselves and we if Gravity Forms adds future exceptions to the function, we don't interfere.